python/ray/autoscaler/autoscaler.py

```diff
@@ -32,6 +33,8 @@
                                  TAG_RAY_NODE_NAME)
 import ray.services as services
 
+logger = logging.getLogger(__name__)
```

Comment:
shall we use log instead of logger? just shorter

Reply:
`logger` is pretty standard.